tests/src/test/scala/packages/samples/CurlTest.scala

```diff
@@ -34,17 +37,27 @@ class CurlTest extends TestHelpers with WskTestHelpers {
 
     it should "Return Could not resolve host when sending no parameter" in {
         val expectedError = "Could not resolve host"
-        val run = wsk.action.invoke(greetingAction, Map())
-        withActivation(wsk.activation, run) {
-            _.response.result.get.toString should include(expectedError)
-        }
+
+        retry(
+            {
+                val run = wsk.action.invoke(greetingAction, Map())
+                withActivation(wsk.activation, run) {
+                    _.response.result.get.toString should include(expectedError)
+                }
+            })
     }
 
     it should "Return the web content when sending the public google as the payload" in {
         val expectedBody = "<HTML>"
-        val run = wsk.action.invoke(greetingAction, Map("payload" -> "google.com".toJson))
-        withActivation(wsk.activation, run) {
-            _.response.result.get.toString should include(expectedBody)
-        }
+        val host = WhiskProperties.getEdgeHost
```

Comment:
The purpose of this curl test is to see if curl action can reach out to the public website, like google. If we use edge node, there is no meaning the test against a local node.